storm-core/src/jvm/backtype/storm/scheduler/resource/RAS_Node.java

```diff
@@ -193,17 +199,21 @@ public void freeAllSlots(Cluster cluster) {
     /**
      * Frees a single slot in this node
      * @param ws the slot to free
-     * @param cluster the cluster to update
      */
-    public void free(WorkerSlot ws, Cluster cluster) {
+    public void free(WorkerSlot ws) {
+        LOG.info("freeing ws {} on node {}", ws, _hostname);
         if (_freeSlots.contains(ws)) return;
         for (Entry<String, Set<WorkerSlot>> entry : _topIdToUsedSlots.entrySet()) {
             Set<WorkerSlot> slots = entry.getValue();
+            double memUsed = this.getMemoryUsedByWorker(ws);
```

Comment:
+1 for this along with all the other method calls in this file. Your argument for using `this` for member variables I can understand (even if I disagree), but using `this` for method calls doesn't convey anything and is just clutter.